libpod/container_internal_common.go

```diff
@@ -2432,6 +2426,85 @@ func (c *Container) generateCurrentUserPasswdEntry() (string, int, int, error) {
 	return pwd, uid, rootless.GetRootlessGID(), nil
 }
 
+// Sets the HOME Env variable with precedence: existing HOME variable, caller home, execUser home
+func (c *Container) setHomeEnvIfNeeded() (string, error) {
```

Comment:
why do you return the home dir but then never use it?

Reply:
It was indeed unnecessary, now only an error is returned.